style.css

```diff
@@ -0,0 +1,187 @@
+/* Media Queries*/
+@media only screen(max-width:500px) {
+    #sidebar{
+        display: none;
+    }
+    .content-wrapper{
+        width: 100%;
+        padding: 0 25px;
+    }
+}
+
+
+body{
```

Comment:
- kindly give a space after line 5
- Please put an indentation before the curly brace in line 6
- Empty space in line 12 should be removed